crypto/fipsmodule/cipher/e_aes.c

```diff
@@ -132,15 +132,19 @@ int GFp_vpaes_set_encrypt_key(const uint8_t *userKey, unsigned bits,
 void GFp_vpaes_encrypt(const uint8_t *in, uint8_t *out, const AES_KEY *key);
 #endif
 
-// XXX: Returns zero on success, violating the return value convention.
-// TODO: should return void anyway, as it can only fail if passed a null pointer.
-int GFp_AES_set_encrypt_key(const uint8_t *user_key, unsigned bits,
-                            AES_KEY *key) {
+// |bits| must be 128 or 256. 192-bit keys are not supported.
+void GFp_AES_set_encrypt_key(const uint8_t *user_key, unsigned bits,
+                             AES_KEY *key) {
   // Keep this in sync with |gcm128_init_gmult_ghash| and |aes_ctr|.
 
+  assert(user_key != NULL);
+  assert(key != NULL);
+  assert(bits == 128 || bits == 256);
+
 #if defined(HWAES)
   if (hwaes_capable()) {
-    return GFp_aes_hw_set_encrypt_key(user_key, bits, key);
+    GFp_aes_hw_set_encrypt_key(user_key, bits, key);
```

Comment:
Please prefix this with `(void)` to make it clear we're ignoring the return value intentionally.